src/components/Checkout/DonationItem.vue

```diff
@@ -307,6 +320,28 @@ export default {
 				this.isCash80Running = true;
 				this.hasCustomDonation = true;
 			}
+
+			// Experiment: CASH-386
+			// eslint-disable-next-line max-len
+			const totalLoansLentQuery = this.apollo.readQuery({
+				query: donationDataQuery,
+			});
+			this.loanHistoryCount = _get(totalLoansLentQuery, 'my.loans.totalCount') || null;
+
+			const nudgeLendingCostExperimentVersion = this.apollo.readQuery({
+				query: experimentAssignmentQuery,
+				variables: { id: 'donation_nudge_lending_cost' },
+			});
+			// eslint-disable-next-line max-len
+			const nudgeLendingCostExperimentVersionString = _get(nudgeLendingCostExperimentVersion, 'experiment.version') || null;
+			if (this.hasLoans && this.loanHistoryCount > 0 && nudgeLendingCostExperimentVersionString === 'variant-a') {
+				this.$kvTrackEvent('basket', 'EXP-CASH-386-Jan2019', 'a');
```

Comment:
Is this variant-a taking the place of 'control'?

Reply:
Yes :+1: